edx_rest_api_client/auth.py

```diff
@@ -48,3 +48,16 @@ def __call__(self, r):
         """ Update the request headers. """
         r.headers['Authorization'] = 'Bearer {}'.format(self.token)
         return r
+
+
+class OidcIdTokenAuth(AuthBase):
```

Comment:
@clintonb I'm with @zubair-arbi here - the only thing that's really similar about the two classes is that they output a JWT auth header, but I don't think it makes sense to combine them.